e2e-tests/common.sh

```diff
@@ -6,60 +6,65 @@
-      fi
-   done
-  echo "OPENMRS SERVER STARTED SUCCESSFULLY"
+    --max-time 20 http://localhost:8099/openmrs/ws/fhir2/R4/Patient 2>/dev/null | cut -d ";" -f 1)
+  until [[ ${contenttype} == "application/fhir+json" ]]; do
+    sleep 60s
+    print_message "WAITING FOR OPENMRS SERVER TO START"
+    contenttype=$(curl -o /dev/null --head -w "%{content_type}\n" -X GET -u admin:Admin123 --connect-timeout 5 \
+      --max-time 20 http://localhost:8099/openmrs/ws/fhir2/R4/Patient 2>/dev/null | cut -d ";" -f 1)
+    ((openmrs_start_wait_time += 1))
+    if [[ ${openmrs_start_wait_time} == 20 ]]; then
+      print_message "TERMINATING TEST AS OPENMRS TOOK TOO LONG TO START"
+      exit 1
+    fi
+  done
+  print_message "OPENMRS SERVER STARTED SUCCESSFULLY"
 
-  cd "${current_path}"
-  echo "STARTING FHIR SINK SERVER"
-  docker-compose -f sink-compose.yml up -d
+  docker-compose -f  docker/sink-compose.yml up -d
   fhir_server_start_wait_time=0
   fhir_server_status_code=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u hapi:hapi --connect-timeout 5 \
     --max-time 20 http://localhost:8098/fhir/Observation 2>/dev/null)
   until [[ ${fhir_server_status_code} -eq 200 ]]; do
-      sleep 1s;
-      echo "WAITING FOR FHIR SERVER TO START"
-      fhir_server_status_code=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u hapi:hapi --connect-timeout 5 \
-            --max-time 20 http://localhost:8098/fhir/Observation 2>/dev/null)
-      ((fhir_server_start_wait_time+=1))
-      if [[ fhir_server_start_wait_time == 10 ]]; then
-        echo "TERMINATING AS FHIR SERVER TOOK TOO LONG TO START"
-        exit 1
-      fi
+    sleep 1s
+    print_message "WAITING FOR FHIR SERVER TO START"
+    fhir_server_status_code=$(curl -o /dev/null --head -w "%{http_code}" -L -X GET -u hapi:hapi --connect-timeout 5 \
+      --max-time 20 http://localhost:8098/fhir/Observation 2>/dev/null)
+    ((fhir_server_start_wait_time += 1))
+    if [[ fhir_server_start_wait_time == 10 ]]; then
+      print_message "TERMINATING AS FHIR SERVER TOOK TOO LONG TO START"
+      exit 1
+    fi
   done
-  echo "FHIR SERVER STARTED SUCCESSFULLY"
+  print_message "FHIR SERVER STARTED SUCCESSFULLY"
+
+  TEST_DIR_FHIR=$(mktemp -d -t analytics_tests__XXXXXX_FHIRSEARCH)
+  TEST_DIR_JDBC=$(mktemp -d -t analytics_tests__XXXXXX_JDBC)
+  JDBC_SETTINGS="--jdbcModeEnabled=true --jdbcUrl=jdbc:mysql://localhost:3306/openmrs --dbUser=mysqluser \
+                 --dbPassword=mysqlpw --jdbcMaxPoolSize=50 --jdbcDriverClass=com.mysql.cj.jdbc.Driver"
 }
 
-function test() {
+function validate_counts() {
   mkdir omrs
   print_message "Finding number of patients, encounters and obs in openmrs server"
   curl -L -X GET -u admin:Admin123 --connect-timeout 5 --max-time 20 \
     http://localhost:8099/openmrs/ws/fhir2/R4/Patient/$1 2>/dev/null >>./omrs/patients.json
-  total_test_patients=$(jq '.total' ./omrs/patients.json)
+    total_test_patients=$(jq '.total' ./omrs/patients.json)
```

Comment:
Why is this indent changed? The old one seems to be correct (also the next two `total_*` lines).